src/coreComponents/constitutive/solid/ElasticIsotropic.hpp

```diff
@@ -0,0 +1,679 @@
+  elasticStrain[5] = m_newStress[k][q][5] / m_shearModulus[k];
+}
+
+
+GEOSX_HOST_DEVICE
+GEOSX_FORCE_INLINE
+void ElasticIsotropicUpdates::smallStrainNoStateUpdate_StressOnly( localIndex const k,
+                                                                   localIndex const q,
+                                                                   real64 const ( & totalStrain )[6],
+                                                                   real64 ( & stress )[6]) const
+{
+  GEOSX_UNUSED_VAR(q);
+      
+  real64 const twoG   = 2 * m_shearModulus[k];
+  real64 const lambda = conversions::BulkModAndShearMod::toFirstLame( m_bulkModulus[k] , m_shearModulus[k] );
+  real64 const vol    = lambda * ( totalStrain[0] + totalStrain[1] + totalStrain[2] );
+
+  stress[0] = vol + twoG * totalStrain[0];
+  stress[1] = vol + twoG * totalStrain[1];
+  stress[2] = vol + twoG * totalStrain[2];
+  
+  stress[3] = m_shearModulus[k] * totalStrain[3];
+  stress[4] = m_shearModulus[k] * totalStrain[4];
+  stress[5] = m_shearModulus[k] * totalStrain[5];
+}
+
+
+GEOSX_HOST_DEVICE
+GEOSX_FORCE_INLINE
+void ElasticIsotropicUpdates::smallStrainNoStateUpdate( localIndex const k,
+                                                        localIndex const q,
+                                                        real64 const ( & totalStrain )[6],
+                                                        real64 ( & stress )[6],
+                                                        real64 ( & stiffness )[6][6] ) const
+{
+  smallStrainNoStateUpdate_StressOnly( k, q, totalStrain, stress);
+  getElasticStiffness( k, stiffness );
+}
+
+
+GEOSX_HOST_DEVICE
+GEOSX_FORCE_INLINE
+void ElasticIsotropicUpdates::smallStrainNoStateUpdate( localIndex const k,
+                                                        localIndex const q,
+                                                        real64 const ( & totalStrain )[6],
+                                                        real64 ( & stress )[6],
+                                                        DiscretizationOps & stiffness ) const
+{
+  smallStrainNoStateUpdate_StressOnly( k, q, totalStrain, stress);
+  stiffness.m_lambda = conversions::BulkModAndShearMod::toFirstLame( m_bulkModulus[k] , m_shearModulus[k] );
+  stiffness.m_shearModulus = m_shearModulus[k];
+}
+
+
+GEOSX_HOST_DEVICE
+GEOSX_FORCE_INLINE
+void ElasticIsotropicUpdates::smallStrainUpdate_StressOnly( localIndex const k,
+                                                            localIndex const q,
+                                                            real64 const ( & strainIncrement )[6],
+                                                            real64 ( & stress )[6]) const
+{
+  smallStrainNoStateUpdate_StressOnly( k, q, strainIncrement, stress); // stress  = incrementalStress
+  LvArray::tensorOps::add< 6 >( stress, m_oldStress[k][q]);            // stress += m_oldStress
```

Comment:
You are adding `m_oldStress`, but in the case of the sedov test problems, the "stress" in a cell is initialized....which is really `m_newStress` now. And `m_oldStress` is not being initialized properly. This is a larger issue than just initialization. I am not sure we want to be carrying around two stresses in the elastic small strain models.

Reply:
Was this the source of the problem we say yesterday?  

Normally I would start by storing the initial stress state in m_oldStress, and it would then be handled naturally through the incremental update process.  I think we should discuss how to revise the NoState model to be the optimized SSLE version we need, and keep the current implementation for the State update for use with derived plasticity models.  Let me know what you think though.

Reply:
We can also just never allocate m_oldStress for elasticity, and work just with m_newStress.  It will be a specialization though.  I prefer to keep the m_oldStress available is SolidBase, even if we don't allocate it. 

How do you handle a re-wind if you have lost history though?

Reply:
More concretely, I think we can remove `stress_n` which is currently registered by the solid solver to save the old stress.  I see your point though about being cautious bringing extra data into kernel when it is not needed.